sass/spinner.scss

```diff
@@ -0,0 +1,81 @@
+$color: #fff;
```

Comment:
agreed but we need consistency across spinner styles, @jnlissy has a card to consolidate all spinner styles so can you sync with her on make sure she is aware of this one as well